embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/network/logging/EmbraceNetworkLoggingService.kt

```diff
@@ -207,9 +130,29 @@ internal class EmbraceNetworkLoggingService(
         }
     }
 
-    private fun storeNetworkCall(callId: String, networkCall: NetworkCallV2) {
+    private fun storeNetworkRequest(networkRequest: EmbraceNetworkRequest) {
         callsStorageLastUpdate.incrementAndGet()
-        sessionNetworkCalls[callId] = networkCall
+
+        //TODO: Why do we want to strip the URL?
```

Comment:
We strip because we want to aggregate all the calls to this base URL without parameters and hash section links (or whenever they are called). I guess the idea is that foo.com/#first and foo.com/#second is close enough that they should be looked at together.